pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -43,10 +43,10 @@ type cadvisorClient struct {
 
 var _ Interface = new(cadvisorClient)
 
-// TODO(vmarmol): Make configurable.
-// The amount of time for which to keep stats in memory.
-const statsCacheDuration = 2 * time.Minute
-const maxHousekeepingInterval = 15 * time.Second
```

Comment:
@vishh was this done for a good reason?  I changed the default equal to the cadvisor default.